src/controllers/suggestion.js

```diff
@@ -0,0 +1,24 @@
+const addSuggestion = require('./../model/queries/addSuggestion');
+const  validateEmail = (email) => {
+  const re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
the same validation in a lot of files
in your next project use some modules like joi or express validator that would make your life easier and also my life :smile: